docs/apis-tools/migration-manuals/migrate-to-camunda-api.md

```diff
@@ -85,25 +85,309 @@ The following conventions apply to all attributes:
 
 <TabItem value='input-adjustments'>
 
-- Filter attribute `assigned (boolean)` removed
-  - Use filter attribute `assignee` with condition `{ "$exists": false }`
-- Filter attribute `assignees (string[])` removed
-  - Use filter attribute `assignee` with condition `{ “$in”: [ “xyz”, ... ] }`
-- Filter attribute `taskDefinitionId` renamed
-  - Use filter attribute `elementId`
-- Filter attribute `candidateGroups (string[])` removed
-  - Use filter attribute `candidateGroup` with condition `{ “$in”: [ “xyz”, ... ] }`
-- Filter attribute `candidateUsers (string[])` removed
-  - Use filter attribute `candidateUser` with condition `{ “$in”: [ “xyz”, ... ] }`
+- Request structure changes as outlined above.
+- `assigned (boolean)` removed
+  - **V2:** Use `assignee` with `{ "$exists": false }`
+- `assignees (string[])` removed
+  - **V2:** Use `assignee` with `{ "$in": [ "xyz", ... ] }`
+- `taskDefinitionId` renamed
+  - **V2:** Use `elementId`
+- `candidateGroups (string[])` removed
+  - **V2:** Use `candidateGroup` with `{ "$in": [ "xyz", ... ] }`
+- `taskVariables` renamed
+  - **V2:** Use `variables`
+- `candidateUsers (string[])` removed
+  - **V2:** Use `candidateUser` with `{ "$in": [ "xyz", ... ] }`
+- `tenantIds (string[])` removed
+  - **V2:** Use `tenantIds` with `{ "$in": [ "xyz", ... ] }`
+- `implementation` removed
+  - **V2:** Only Camunda user tasks returned
```

Comment:
Note that the term "Camunda user task" will go out in 8.7 with https://github.com/camunda/camunda-docs/pull/4723.

Reply:
Thanks @christinaausley, just to understand, is there any action to take now (change)? Or this term should be changed in the future?

Reply:
No, this is changing to "Camunda user task" with that PR, which goes out for `alpha-3`. Since this PR also goes out with `alpha-3`, this should be fine 😄 Just didn't want this to go into the docs now if users don't know what the term is yet.